setup/aix61/README.md

```diff
@@ -0,0 +1,22 @@
+# io.js Build AIX61 Setup
+
+For setting up an AIX 61 box
+
+```text
+Host test-ibm-aix61-ppc64-1:18822
+  HostName cloud.siteox.com
+  User root
+  This machine is a temporary one until we get additional resources at osusol. It is some
```

Comment:
Could you move this text out of the code? The pattern seems to be using code for copy/paste into ssh config.